.travis.yml

```diff
@@ -30,6 +30,7 @@ matrix:
     - php: '7.0'
```

Comment:
I think we should also drop support for PHP 5.6 and 7.0, as 7.0 has been EOL'ed and 5.6 will be at the end of the year.